_posts/2020-06-04-App Service Continuous Deployment for Windows Containers with GitHub Actions.md

```diff
@@ -0,0 +1,403 @@
+---
+title: "Continuous Deployment for Windows Containers with GitHub Actions"
+author_name: "Jeff Martinez"
+toc: true
+toc_sticky: true
+tags:
+    - dotnet
+    - windows containers
+---
+
+Github Actions enables you to easily automate any part of your development workflow. GitHub Actions are defined as YAML files in the `.github/workflows` directory of your repository. The workflows are triggered by an event, such as a push to a specific branch, a comment on a Pull Request, or on a CRON schedule.
+
+In this article, we will use GitHub Actions for Azure to deploy a Windows Container application to App Service. The sample application is already configured to be used in a Windows Container app, pushed to a registry, and deployed to App Service. Of course, you can use this guide to add the correct deployment tasks to work with your own applications.
+
+## Sample Application
+
+> If you would like to get started with your own application, you can skip to the next section.
+
+The sample application is a simple task-tracking app built with .NET Framework using Azure SQL for storage.  The project has a workflow file, *main.yaml*, that is set up for continuous deployment. You have your choice of using Azure Container Registry (ACR) or Docker Hub for your registry needs (the difference in syntax will be explained below).
+
+Find the full repository samples for [.NET Framework](https://aka.ms/dotnetframeworkdeployment) and [.NET Core](https://aka.ms/dotnetcoredeployment) at these highlighted links.
+
+1. Clone the repository
+1. Add in the necessary GitHub secrets so actions knows where to connect to your Azure resources and committing your changes to the master branch will trigger the build.  
```

Comment:
I think we should cut this. I think you're alluding to the following steps, but as a reader I think I'm supposed to take action here, but it's to vague for me to know what to do.

Reply:
Are you saying that we should ditch the sample links too?  We want the samples available so customers can clone them, plug in their info, and run the cd scenario.  Maybe I clarify that they have two paths here?  Meaning if they don't use the samples, they follow the blog as a walkthrough to do it themselves with their own apps...